client/components/Medical-Contact/ContactPage.vue

```diff
@@ -0,0 +1,51 @@
+<!-- Page for medical contacts management -->
+<!-- User should be authenticated in order to see this page -->
+
+<template>
+  <main>
+    <hr>
+    <section>
+      <header>
+        <h2>Health Book</h2>
+      </header>
+      <!-- <button>
+        Add contact
```

Comment:
I would love to have this button toggle the visibility of the form in the future, but doesn't need to be implemented for beta.